cmd/infracost/comment.go

```diff
@@ -119,6 +111,20 @@ func buildCommentOutput(cmd *cobra.Command, ctx *config.RunContext, paths []stri
 		}
 	}
 
+	commentPath, _ := cmd.Flags().GetString("comment-path")
+	if commentPath != "" {
+		commentData, err := output.LoadCommentData(commentPath)
+		if err != nil {
+			return nil, fmt.Errorf("Error loading %s used by --comment-path flag. %s", commentPath, err)
+		}
+
+		return &CommentOutput{
+			Body:    commentData,
+			HasDiff: combined.HasDiff(),
+			ValidAt: &combined.TimeGenerated,
+		}, nil
```

Comment:
seems like we should keep the error return logic (policy & governance).  These will be important so github actions runner can correctly fail if governance comment fails